lib/segment/src/index/field_index/full_text_index/text_index.rs

```diff
@@ -312,13 +312,26 @@ impl FullTextIndex {
         }
     }
 
+    /// Tries to parse a phrase query. If there are any unseen tokens, returns `None`
```

Comment:
> If there are any unseen tokens, returns `None`

Is this correct?

Reply:
Yes, if a token hasn't been indexed, it means the query will return 0 results in this segment